common/lib/xmodule/xmodule/modulestore/tests/test_mongo.py

```diff
@@ -191,15 +204,18 @@ def test_mongo_modulestore_type(self):
     def test_get_courses(self):
         '''Make sure the course objects loaded properly'''
         courses = self.draft_store.get_courses()
-        assert_equals(len(courses), 6)
+        assert_equals(len(courses), 7)  # pylint: disable=no-value-for-parameter
```

Comment:
Hrm. This needs more investigation. If I run this specific test there are 6, if I run the whole suite then there are 7. Seems like probably another test run isn't cleaning something up in the mongo database....